README.md

```diff
@@ -69,6 +70,11 @@ We are present in nearly the complete round-up of social networks. Follow/friend
 </p>
 
 
+## Knowledge Base
+
+Find documentation, instructions, and helpful references in our [Terasology Knowledge Base](http://terasology.org/Terasology/#/), formerly known as the Terasology Engine wiki.
```

Comment:
This is the first time that I've heard it called the "Terasology Knowledge Base". Does it imply that the documentation there is player-facing now, as well as developer-facing? The wiki was predominantly a developer resource, with a few player pages for troubleshooting.

Reply:
I called it that because @skaldarnar suggested that on Sunday and I think it's a good name.
I think it makes sense that the documentation is addressing different audiences (as also mentioned in #5148).
Having one place for all our documentation (irrespective of whether it's for players, devs, or maintainers) allows for easier management and overview as well as reduces siloing the different audience groups. They will have different ways of getting to that central documentation space for sure - e.g coming from the website (players or very new devs), coming from the game (players and devs), coming from github (devs), etc. - and will also have different sections within, but I think we can create a more homogenous feeling which makes it easier to cross the boundaries, e.g. from being a player to becoming a dev or from being a dev to becoming a maintainer.